templates/zerver/register.html

```diff
@@ -74,104 +79,108 @@ <h1>You&rsquo;re almost there.</h1>
                             {% endfor %}
                         {% endif %}
                     </div>
-                    <div class="help-box margin-top">
-                        {{ _("The address you'll use to log in to your organization.") }}
+                    <div class="help-text">
+                        {{ _("The URL users will use to access the new organization.") }}
                     </div>
                 </div>
                 {% endif %}
             </section>
 
             <section class="user-registration">
```

Comment:
I think we should replace both sections with fieldsets. Wrapping fieldsets in sections doesn't make much sense semantically since a fieldset already defines a section of a form.